Spark Guides: Content Style (#4012)
sparkdesignsystem/spark-design-system: src/pages/using-spark/guides/content-style.mdx

---
title: Content Style
---

import {
  SprkStack,
  SprkStackItem,
  SprkButton,
  SprkFlag,
  SprkHeading,
  SprkLink,
  SprkTextInput,
  SprkCard,
  SprkIcon,

>   SprkDivider,

Reviewer: We don't actually use all of these in this file. I would remove any that we aren't utilizing.

Author: @guern1kn removed unused ones

} from '@sparkdesignsystem/spark-react';

> <h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure ">

Reviewer: ```suggestion
<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
```

  Content Style
</h1>

UX Writers, designers and developers can use the **Content Style Guides** as a instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.

<SprkDivider element="span" additionalClasses="sprk-u-mvn" />

### Table Of Contents

<ul>
  <li>
    <a href="#capitalization">Capitalization</a>
  </li>

>   <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
>     <li>
>       <a href="#sentence-case">Sentence Case</a>
>     </li>
>     <li>
>       <a href="#start-case">Start Case</a>
>     </li>
>   </ul>

Reviewer 2: ```suggestion
  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
    <li>
      <a href="#start-case">Start Case</a>
    </li>
    <li>
      <a href="#sentence-case">Sentence Case</a>
    </li>
  </ul>
```

These should be in the same order that they appear in the page.

Author: Thank you for catching this @bobwise , updated the TOC

#### Usage

- Headers that are descriptive or functional
- All subheads regardless of whether the primary header is title or sentence case
- All buttons labels (Primary, Secondary and Tertiary Buttons)
- Menu items
- Chart and table headers
- Referring to the name of a page

#### Examples

- Homeowners Insurance Info
- How To Buy A Home
- Lock Your Rate And Save
- Check Your To Do List

<SprkDivider element="span" additionalClasses="sprk-u-mvn" />

### Sentence Case

Sentence case capitalizes the first word of the sentence and proper nouns.

#### Usage

- Emotional and Impactful headers
- Special instances, where there may be two primary headers on a page
- Body copy
- Email subject lines and pre-headers
- Use end punctuation when the list item is a complete sentence

#### Examples

- We’re here to help!
- If you want the lowest payment possible, you might want to look at a 30-year fixed.
- Lock your rate and save.
- Interest rates are on the rise.
- Welcome to Rocket Mortgage.

<SprkDivider element="span" additionalClasses="sprk-u-mvn" />

## Quotation Marks

Use opening and closing quotes to show the start and endpoint of quoted content.

> #### Usage

Reviewer 2: This triggers an accessibility error in [Axe Dev Tools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US): "Heading levels should only increase by one".

We can either change Usage and Examples to `###` under Quotation Marks, or we can replace them with this in order to maintain their current style:

```html
<h3 class="sprk-b-TypeDisplayFour docs-b-h4 sprk-u-mbm sprk-u-Measure">
  Usage
</h3>
```

Author: @bobwise updated "usage" and "examples" headers to match above style

---
title: Content Style
---

import {
  SprkHeading,

>   SprkLink,

Reviewer: These aren't used, so we just need the SprkDivider imported.

<SprkDivider element="span" additionalClasses="sprk-u-mvn" />

### Table Of Contents

<ul>
  <li>
    <a href="#capitalization">Capitalization</a>
  </li>
  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
    <li>
      <a href="#start-case">Start Case</a>
    </li>
    <li>
      <a href="#sentence-case">Sentence Case</a>
    </li>
  </ul>
  <li>
    <a href="#quotation-marks">Quotation Marks</a>
  </li>
  <li>
    <a href="#accessibility">Accessibility</a>
  </li>
</ul>

<SprkDivider element="span" additionalClasses="sprk-u-mvn" />

## Capitalization

Use for visual presentation of pieces of text to make it easy for users to read.

> As a general rule, use **start case** for **headers**, use either **start case** or **sentence case** for **drop down options** and **labels**, depending on the context.

### Start Case

Capitalize the first letter of each word, regardless of length or part of speech. When using start case, avoid using end punctuation other than question marks.

> <h3 class="sprk-b-TypeDisplayFour docs-b-h4 sprk-u-mbm sprk-u-Measure">

Reviewer: Can remove docs-b-h4, as it doesn't do anything. Remove throughout the file.

---
title: Content Style
---

import {
  SprkDivider,
} from '@sparkdesignsystem/spark-react';

<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
  Content Style
</h1>

> UX Writers, designers and developers can use the **Content Style Guides** as a instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.

Reviewer 3: Is "content style guides" supposed to be a link? the word *the* implies this is a document somewhere.

Reviewer 3: is "the" supposed to be "this" ?

Reviewer: I agree with RV. We're on the page, so it feels like it should be "this Content Style Guide".

> UX Writers, designers and developers can use this **Content Style Guides** as a instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.

Reviewer: ```suggestion
UX Writers, designers and developers can use this **Content Style Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
```

<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">
  Usage
</h3>

- Headers that are descriptive or functional
- All subheads regardless of whether the primary header is title or sentence case
- All buttons labels (Primary, Secondary and Tertiary Buttons)
- Menu items
- Chart and table headers
- Referring to the name of a page

<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">
  Examples
</h3>

- Homeowners Insurance Info
- How To Buy A Home
- Lock Your Rate And Save
- Check Your To Do List

<SprkDivider element="span" additionalClasses="sprk-u-mvn" />

### Sentence Case

Sentence case capitalizes the first word of the sentence and proper nouns.

<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">
  Usage
</h3>

- Emotional and Impactful headers
- Special instances, where there may be two primary headers on a page
- Body copy
- Email subject lines and pre-headers
- Use end punctuation when the list item is a complete sentence

<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">
  Examples
</h3>

- We’re here to help!

> - If you want the lowest payment possible, you might want to look at a 30-year fixed.

Reviewer 3: is it industry knowledge to leave out the word "loan" at the end of this sentence? 
It might be good to ask or to just be clear and say 30-year fixed loan.
